cmd/main.go

```diff
@@ -18,6 +18,7 @@ package main
 
```

Comment:
Similarly IMO `main.go` at root level seems more apt, I am unsure about moving it to `cmd`
Ref: https://sdk.operatorframework.io/docs/overview/project-layout/#golang

Reply:
this is how it is generated in 1.34 https://github.com/operator-framework/operator-sdk/tree/v1.34.x/testdata/go/v4/memcached-operator/cmd